snmp/datadog_checks/snmp/cmd.py

```diff
@@ -0,0 +1,134 @@
+# (C) Datadog, Inc. 2020-present
```

Comment:
Also, should this file be named a more explicit `commands.py`? ^_^

Reply:
Sure :).